frontend/src/components/InfoWindow/InfoWindow.jsx

```diff
@@ -1,15 +1,35 @@
+import React from "react";
+import DOMPurify from "dompurify";
+
 /**
  * The react component for the information window for each node
  * in the concept tree.
+ * Note: dangerouslySetInnerHTML property used here becomes safe if
+ * sanitization is done to the incoming htmlInfo using DOMPurify
  */
 
-import React from "react";
-
-export const InfoWindow = ({ info }) => {
+export const InfoWindow = ({ info, linkHoverHandler }) => {
+  const highlightRelatedNode = (tag) => {
+    const tagName = tag.target.tagName;
+    if (tagName === "A") {
```

Comment:
What does tagName A stand for? Can we name it a more informative name?

Reply:
changed